tests/generators/forks/main.py

```diff
@@ -1,40 +1,38 @@
 from importlib import reload
 from typing import Iterable
 
-from eth2spec.test.context import LIGHTCLIENT_PATCH
+from eth2spec.test.context import PHASE0, LIGHTCLIENT_PATCH, MINIMAL, MAINNET
 from eth2spec.config import config_util
 from eth2spec.test.lightclient_patch.fork import test_fork as test_altair_forks
 from eth2spec.phase0 import spec as spec_phase0
+from eth2spec.lightclient_patch import spec as spec_lightclient_patch
 
 from eth2spec.gen_helpers.gen_base import gen_runner, gen_typing
 from eth2spec.gen_helpers.gen_from_tests.gen import generate_from_tests
 
 
-pre_specs = {
-    LIGHTCLIENT_PATCH: spec_phase0,
-}
-
-
-def create_provider(fork_name: str, tests_src, config_name: str) -> gen_typing.TestProvider:
+def create_provider(tests_src, config_name: str) -> gen_typing.TestProvider:
 
     def prepare_fn(configs_path: str) -> str:
         config_util.prepare_config(configs_path, config_name)
-        reload(pre_specs[fork_name])
+        reload(spec_phase0)
+        reload(spec_lightclient_patch)
         return config_name
 
     def cases_fn() -> Iterable[gen_typing.TestCase]:
         return generate_from_tests(
-            runner_name='forks',
-            handler_name='core',
+            runner_name='fork',
+            handler_name='fork',
```

Comment:
Following the pattern of `finality` test vector output, I made it follow the folder name and the `test_*` filename.